corehq/apps/hqwebapp/crispy.py

```diff
@@ -24,6 +26,13 @@ class HQFormHelper(FormHelper):
 
     def __init__(self, *args, **kwargs):
         super(HQFormHelper, self).__init__(*args, **kwargs)
+        from corehq.apps.hqwebapp.utils.bootstrap import get_bootstrap_version, BOOTSTRAP_5
+        bootstrap_version = get_bootstrap_version()
+        use_bootstrap5 = bootstrap_version == BOOTSTRAP_5
+        if use_bootstrap5:
```

Comment:
I was going to make a nitpick comment about the `use_bootstrap5` being unnecessary given it is used once, but am in favor of using it for the sake of consistency with other usages of it 👍🏻